### ISSUE TITLE
Following link in splash, installed from docker hungs it up and eating up all mem.

### ISSUE THREAD

**Issue by yadalik** [scrapinghub/splash#849]
1. Installing Splash with docker by command:
`docker run -p 8050:8050 scrapinghub/splash`
on Ubuntu 18.04 TLS (installed inside VMWare), with 8Gb RAM
2. Opening address [http://splash_ip:8050](http://splash_ip:8050)
3. Inserting this link in "Render me" field:
[https://www.healthgrades.com/usearch?what=Pain%20Management&where=Alabaster%2C%20AL&city=Alabaster&state=AL&pt=33.245079%2C%20-86.816223&pageNum=1](https://www.healthgrades.com/usearch?what=Pain%20Management&where=Alabaster%2C%20AL&city=Alabaster&state=AL&pt=33.245079%2C%20-86.816223&pageNum=1)
4. Everything hangs up, page says "rendering, please wait", htop shows one of cores 100% utilized and RAM slowly increasing, until OOM starts to killing processes, in docker  output I have this:
```
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
process 1: D-Bus library appears to be incorrectly set up; failed to read machin                                                           e uuid: UUID file '/etc/machine-id' should contain a hex string of length 32, no                                                           t length 0, with no other text
See the manual page for dbus-uuidgen to correct this issue.
qt.network.ssl: QSslSocket: cannot resolve SSLv2_client_method
qt.network.ssl: QSslSocket: cannot resolve SSLv2_server_method
```

**Comment by lucywang000** [comment]
This could be a webkit problem - there is a work in progress to add chrominum engine support https://github.com/scrapinghub/splash/pull/867/

**Comment by yadalik** [comment]
BTW output in 4. is useless for this problem, cause I have it also for google.com, but with google.com everything loading. With mentioned address this problem remains. Also, I've tried run with verbose settings, `docker run -it -p8050:8050 scrapinghub/splash -v2` last message seems fine, and then complete stop (+100% CPU+Mem usage increasing till OOM killer):
`2019-02-15 07:17:41.383585 [network-manager] Finished downloading https://www.medtargetsystem.com/javascript/blu3print.jv.js?3.18.0`

**Comment by BinhMinhs10** [comment]
type like below:
sudo docker run --net="host" -p 8050:8050 -p 5023:5023 scrapinghub/splash -v4

**Comment by yadalik** [comment]
tl;dr: not helped in this case.

Hi, did you tried to set this URL in "Render me" field and it worked: https://www.healthgrades.com/usearch?what=Pain%20Management&where=Alabaster%2C%20AL&city=Alabaster&state=AL&pt=33.245079%2C%20-86.816223&pageNum=1 ?

Maybe I'm doing something wrong, but when I type:

` docker network create my-net`

(and even `sudo sysctl net.ipv4.conf.all.forwarding=1` and `sudo iptables -P FORWARD ACCEPT`)

`docker run --net="my-net" -p 8050:8050 -p 5023:5023 scrapinghub/splash -v4`

I'm still getting this last string and hung's up with eating RAM and CPU until OOM-killer comes:
`2019-04-02 04:49:39.648374 [network-manager] Finished downloading https://www.medtargetsystem.com/javascript/blu3print.jv.js?3.23.1`

If I understang correctly, you mean that creating user defined bridge as described in https://docs.docker.com/network/bridge/ should help, cause "Containers connected to the same user-defined bridge network automatically expose all ports to each other, and no ports to the outside world. This allows containerized applications to communicate with each other easily, without accidentally opening access to the outside world."
